.idea/misc.xml

```diff
@@ -6,6 +6,7 @@
         <entry key="app/src/main/res/layout/activity_photo_gallery.xml" value="0.37604166666666666" />
         <entry key="app/src/main/res/layout/fragment_photo_gallery.xml" value="0.375" />
         <entry key="app/src/main/res/layout/list_item_gallery.xml" value="0.375" />
+        <entry key="app/src/main/res/menu/fragment_photo_gallery.xml" value="0.37604166666666666" />
```

Comment:
you should add this file to .gitignore :)

Reply:
ok, thanks!